tools/bin/ci_credentials.sh

```diff
@@ -1,26 +1,150 @@
 #!/usr/bin/env bash
 
 . tools/lib/lib.sh
+. tools/lib/gcp-token.sh
 
 set -e
 
+COMMAND_NAME=$1
```

Comment:
```suggestion
# This script loads secrets from Google Secrets Manager into the relevant connector's working directory
# If a secret is available in both Github and GSM, then the GSM secret is used, otherwise Github. 
COMMAND_NAME=$1
```